app/src/main/java/com/lamdangtung/calculatorapp/MainActivity.kt

```diff
@@ -0,0 +1,292 @@
+package com.lamdangtung.calculatorapp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import com.lamdangtung.calculatorapp.databinding.ActivityMainBinding
+import java.util.*
+
+class MainActivity : AppCompatActivity(), View.OnClickListener {
+
+    companion object Constants {
+        private const val EXTRA_TEXT_FORMUlA: String = "com.lamdangtung.calculatorapp.EXTRA_TEXT_FORMULA"
+        private const val EXTRA_TEXT_RESULT: String = "com.lamdangtung.calculatorapp.EXTRA_TEXT_RESULT"
```

Comment:
```suggestion
        private const val EXTRA_TEXT_FORMUlA = "com.lamdangtung.calculatorapp.EXTRA_TEXT_FORMULA"
        private const val EXTRA_TEXT_RESULT = "com.lamdangtung.calculatorapp.EXTRA_TEXT_RESULT"
```
với để companion object xuống cuối cho dễ nhìn